app/models/concerns/host_searching.rb

```diff
@@ -5,8 +5,12 @@ module HostSearching
   extend ActiveSupport::Concern
 
   included do
-    scoped_search on: %i[display_name os_major_version os_minor_version]
+    scoped_search on: %i[id display_name], only_explicit: true
     scoped_search on: :display_name, rename: :name
```

Comment:
Everything is now `only_explicit: true` except name so we don't run into this again.